### PR TITLE
Document cluster configuration and monitoring (RIPD-732)

### PR DESCRIPTION
**Pull request by JoelKatz**
Documentation changes only

### CODE REVIEW

File: src/ripple/app/peers/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+## Configuration ##
+
+A server's public key can be determined from the output of the `server_info`
+command.  The key is in the `pubkey_node` value, and is a text string
+beginning with the letter `n`.  The key is maintained across runs in a
+database.
+
+Cluster members are configured in the `rippled.cfg` file under
+`[cluster_nodes]`.  Each member should be configured on a line beginning
+with the node public key, followed optionally be a space and a friendly
```
Comment (sublimator) [inline src/ripple/app/peers/README.md]: `followed optionally be a space`

s/be/by/

**Comment by vinniefalco** [comment]
This should probably go in `src/ripple/overlay/README.md`

**Comment by JoelKatz** [comment]
It doesn't really seem to fit in with the other discussion at `src/ripple/overlay/README.md`. At least, not to me.

**Comment by rec** [comment]
Ping?  Can we rebase?

**Pull request closed without merging**